WordPress/gutenberg
# [RNMobile] Fix a11y issue in Cover block

## Description


Fixes: https://github.com/wordpress-mobile/gutenberg-mobile/issues/2699

That PR is introducing fixes for a11y related to Cover block such as:

* selecting the Cover block gives you the option to choose an image
* selecting a edit button over the Cover with an image (to choose different image or video, see the screenshot below) gives user hints about its functionality

<img width="841" alt="Screenshot 2021-02-02 at 11 48 55" src="https://user-images.githubusercontent.com/22746080/106589761-abd95900-654c-11eb-8e4d-161085b833fc.png">

## How has this been tested?

1. Enable VoiceOver
2. Add new post
3. Add `Cover` block
4. Select a `Cover` block

** Expect picker appears and you can choose one of the options **

5. Add a photo
6. Select a edit button over the image

** Expect to hear the hints about buttons functionality **





## Screenshots 

## Types of changes

Bug fix / Accessibility improvements






## Checklist:
